### PR TITLE
Updated plist doctype for alpha 7

### PR DESCRIPTION
**Pull request by thecodechef**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -9,5 +9,5 @@ module.exports = {
   '1.1': '<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">',
   'basic': '<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML Basic 1.1//EN" "http://www.w3.org/TR/xhtml-basic/xhtml-basic11.dtd">',
   'mobile': '<!DOCTYPE html PUBLIC "-//WAPFORUM//DTD XHTML Mobile 1.2//EN" "http://www.openmobilealliance.org/tech/DTD/xhtml-mobile12.dtd">',
-  'plist': '<!DOCTYPE plist PUBLIC "-//APPLE Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">'
+  'plist': "<?xml version=\"1.0\" encoding=\"utf-8\" ?>\n<!DOCTYPE plist PUBLIC \"-//APPLE Computer//DTD PLIST 1.0//EN\" \"http://www.apple.com/DTDs/PropertyList-1.0.dtd\">"
```
Comment (TimothyGu) [inline index.js:12]: Why did you change single quotes to double?

Comment (thecodechef) [inline index.js:12]: because if you don't the escaped characters will mess up. and whenever i tried too do to doctype xml doctype plist it never would look how i wanted it to unless the devs got together and fix a certain bug that the doctypes land the top line right after each other when they need to on their own separate line.

Comment (thecodechef) [inline index.js:12]: Added another pull-request just close this one ok
